packages/server/graphql/types/User.ts

```diff
@@ -148,9 +145,10 @@ const User = new GraphQLObjectType<any, GQLContext, any>({
     isConnected: {
       type: GraphQLBoolean,
       description: 'true if the user is currently online',
-      resolve: (source) => {
-        const {connectedSockets} = source
-        return Array.isArray(connectedSockets) && connectedSockets.length > 0
+      resolve: async ({id: userId}) => {
+        const redis = getRedis()
+        const userPresence = await redis.lrange(`presence:${userId}`, 0, -1)
```

Comment:
`LLEN` is a little cheaper since we don't need to read the values, just the length.